examples/webpack.config.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
   ],
   resolve: {
     extensions: ['.tsx', '.ts', '.js', '.css'],
-    symlinks: true
+    symlinks: false
```

Comment:
Why is this changed here?

Reply:
It's not a change from this PR, just accidentally commited it, now should be correct